mysql-test/CMakeLists.txt

```diff
@@ -87,7 +87,6 @@ IF(NOT WITH_XTRABACKUP)
   IF(DEBUG_EXTNAME)
     SET(DEBUG_SERVER "--debug-server")
   ENDIF()
-<<<<<<< HEAD
```

Comment:
To avoid future conflicts? can you can just use upstream mysql-test/CMakeLists.txt and just update main CMakeLists.txt

> -IF(NOT WITHOUT_SERVER)
+IF(NOT WITHOUT_SERVER AND NOT WITH_XTRABACKUP)
   ADD_SUBDIRECTORY(mysql-test)

Reply:
Done